stdlib/public/core/StringBridge.swift

```diff
@@ -210,53 +187,77 @@ private func _getCocoaStringPointer(
   return .none
 }
 
+@_effects(releasenone)
+public //SPI(Foundation)
+func _bridgeConstantCocoaString(
+  _ cocoaString: __owned AnyObject, _ length: Int) -> String {
+  let (fastUTF8, isASCII): (Bool, Bool)
+  switch _getCocoaStringPointer(cocoaString) {
+  case .ascii(_): (fastUTF8, isASCII) = (true, true)
+  case .utf8(_): (fastUTF8, isASCII) = (true, false)
+  default:  (fastUTF8, isASCII) = (false, false)
+  }
+  
+  return String(_StringGuts(
+    cocoa: cocoaString,
+    providesFastUTF8: fastUTF8,
+    isASCII: isASCII,
+    length: length))
+}
+
+@_effects(releasenone)
+public //SPI(Foundation)
+func _bridgeUnknownCocoaString(
+  _ cocoaString: __owned AnyObject, _ length: Int) -> String {
+ return _bridgeConstantCocoaString(cocoaString, length)
```

Comment:
Again, this reads to me that _bridgeConstantCocoaString is narrowly defined to only constant strings, so I would expect the opposite direction.

Reply:
Sounds good